tests/E2E Tests/TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -299,5 +301,45 @@ private static async Task CreateGraphClientAndAssert(TokenAcquirerFactory tokenA
             Assert.NotNull(result.AccessToken);
         }
     }
+
+    public class AcquireTokenManagedIdentity 
+    {
+        [OnlyOnAzureDevopsFact]
+        //[Fact]
+        public async Task AcquireTokenWithManagedIdentity_UserAssigned()
+        {
```

Comment:
I know we don't have this consistently in our tests, but yours in a nice example to have it split up w/
//Arrange
//Act
//Assert

labels on the different parts.